packages/gatsby/package.json

```diff
@@ -141,6 +141,7 @@
     "@babel/preset-react": "7.0.0-beta.42",
     "@babel/runtime": "7.0.0-beta.42",
     "cross-env": "^5.1.4",
+    "eslint-plugin-react": "^7.8.2",
```

Comment:
This is a duplicate?

Reply:
It seems so. `yarn upgrade-interactive` seems to have played tricks on me. 😅